terraform/testcases/otlp_metric_adot_operator/otconfig.tpl

```diff
@@ -0,0 +1,25 @@
+extensions:
+      pprof:
+        endpoint: 0.0.0.0:1777
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:${grpc_port}
+
+    processors:
+      batch:
+
+    exporters:
+      logging:
+        loglevel: debug
+      awsemf:
+        region: '${region}'
+
+    service:
+      pipelines:
+        metrics:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [logging,awsemf]
+      extensions: [pprof]
```

Comment:
it looks the yaml content is not well formatted as well in this PR

Reply:
I changed the formatting to match the otlp_trace_adot_operator. It works now.

Reply:
I think the formatter is putting this file at the begging of line thus we need a double indent. https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/eks/adot-operator/adot_collector_deployment.tpl https://github.com/aws-observability/aws-otel-test-framework/blob/terraform/terraform/testcases/otlp_trace_adot_operator/otconfig.tpl